### PR TITLE
Add go batches

### PR DESCRIPTION
**Pull request by marstr**
The Azure REST API Specs portion of onboarding the Go SDK to be generated by the `Swagger -> SDK` bot.

### CODE REVIEW

**Comment by marstr** [comment]
This is still in progress, and shouldn't be merged until I remove the `DoNotMerge` tag.

**Comment by marstr** [comment]
@lmazuel I believe this is all ready for your consideration now. Let me know, and we'll get this merged in. :)

File: specification/compute/resource-manager/readme.md [hunk 1]
```diff
@@ -242,6 +243,20 @@ go:
   clear-output-folder: true
 ```
 
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-compute-2017-03
+  - tag: package-container-service-2017-01
```
Comment (lmazuel) [inline specification/compute/resource-manager/readme.md]: Current recommendation is to ship "containerservice" outside of compute. These files are even deprecated, the new can be found here:
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/containerservices/resource-manager

File: specification/eventgrid/data-plane/readme.md [hunk 2]
```diff
@@ -52,6 +52,17 @@ input-file:
...
 
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-go
```
Comment (lmazuel) [inline specification/eventgrid/data-plane/readme.md]: Please add python here! :)

File: specification/compute/resource-manager/readme.md (hunk 1 above)
Comment (marstr) [reply specification/compute/resource-manager/readme.md]: Ah, for the moment I am publishing [the union of API Versions that are present in both locations.](https://github.com/Azure/azure-sdk-for-go/tree/a49674fdba11173ce3bc934d020634e3366464de/services/containerservice/mgmt) There seems to be no overlap, so at the moment we don't have to choose how to resolve any conflicts.

File: specification/azsadmin/resource-manager/commerce/readme.md [hunk 3]
```diff
@@ -39,6 +39,18 @@ input-file:
...
 
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
```
[the remark this answers is not in the document]
Comment (marstr) [reply specification/azsadmin/resource-manager/commerce/readme.md]: Fixed :)

**Comment by marstr** [comment]
Anything else that needs to be changed before this is merged, @lmazuel? :)

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
Manual testing content is good, let's merge!!!!

**Pull request merged**